structuredevent-model/src/main/java/com/sequenceiq/cloudbreak/structuredevent/event/OperationDetails.java

```diff
@@ -101,6 +107,10 @@ public void setTimestamp(Long timestamp) {
         this.timestamp = timestamp;
     }
 
+    public String getUTCDateTime() {
+        return ZonedDateTime.ofInstant(Instant.ofEpochMilli(timestamp), ZoneOffset.UTC).format(DATE_TIME_FORMATTER);
```

Comment:
Shouldn't we add an extra null check to it?

Reply:
We can add, there wasn't in the previous implementation, but it's better to check if it is null or not

Reply:
Right, today morning we already run 3 different NPEs in different places.